### PR TITLE
Contact and footer section

### PR DESCRIPTION
**Pull request by kelomo2502**
For this milestone of the project:

- I built the contact form in the footer
- I followed the list of best practices for HTML and CSS
- I made sure there are no linter errors
- I used the correct GitHub flow
- I made sure I documented my work in a professional way
-  Name field accepts up to 30 chars.

- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
- I used flexbox to layout elements on the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,215 +1,267 @@
...
-        <ul class="footer-socials">
+          <ul class="footer-socials">
             <li><a href="#"><img src="./images/github-new.png" alt="github"></a></li>
             <li><a href="#"><img src="./images/linkedin-new.png" alt="linkedin"></a></li>
             <li><a href="#"><img src="./images/angelistic-new.png" alt="angelistic"></a></li>
             <li><a href="#"><img src="./images/twitter-new.png" alt="twitter"></a></li>
             <li><a href="#"><img src="./images/medium-new.png" alt="medium"></a></li>
-        </ul>
+          </ul>
     </footer>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly add font weight to the paragraph as per the Figma design.
- [ ] Kindly add `hr` (horizontal line) before the social media icons as given in the Figma design.
- [ ] Kindly remove the unnecessary spacing at the bottom of the social media icons.

**_Find the required changes in the video given_**:

https://user-images.githubusercontent.com/70269507/165586965-f1047f18-3935-4c7a-8ac0-984108d58c07.mov

- [ ] Kindly add button interactions such as hover, active (when it's pressed), disabled and enabled as per the Figma design given:

Screenshot 2022-04-27 at 8 28 11 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @kelomo2502, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good PR Description and title
✔️ All linters passing
✔️ Correct implementation of the contact form 

## Required 
- [x] Kindly update your readme as per the project you are working on.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kelomo2502) [reply index.html]: Please I would appreciate if you could expatiate on specific update  I need to make to my README.md as per the project.

**Comment by kelomo2502** [comment]
> ## Required Changes ♻️
> Hi @kelomo2502, This is @sja-thedude your code reviewer.
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> 768070f3-bc9a-4aba-b8bc-25793e00ea1e_text
> 
> ## Highlights
> ✔️ Good PR Description and title ✔️ All linters passing ✔️ Correct implementation of the contact form
> 
> ## Required
> * [x]  Kindly update your readme as per the project you are working on.
> * [ ]  _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kelomo2502** [comment]
Please I would appreciate if you could expatiate on specific update  I need to make to my README.md as per the project.

**Comment by sja-thedude** [comment]
> Please I would appreciate if you could expatiate on specific update  I need to make to my README.md as per the project. 

Hi @kelomo2502! I mean to say like add something like your description example-:
This project is part of Module 1 Week 2 Microverse curriculum where we are working on making our portfolio. In this branch I have worked with my coding partner on building contact section and footer. Where I have used formspree for contact form to be submitted. I have followed the design, color and template according to figma design.

You can add description in readme like this for all the coming projects too.

**Comment by kelomo2502** [comment]
> 

Oh! Thank you. I really appreciate your prompt response. will implement the required changes and submit for a re-review. Cheers!

**Comment by kelomo2502** [comment]
Thank you . I appreciate it.

**Review by keroloussamy — APPROVED** [review]
Hi @kelomo2502, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**